clippy_lints/src/booleans.rs

```diff
@@ -404,6 +412,6 @@ impl<'a, 'tcx> Visitor<'tcx> for NonminimalBoolVisitor<'a, 'tcx> {
         }
     }
     fn nested_visit_map<'this>(&'this mut self) -> NestedVisitorMap<'this, 'tcx> {
-        NestedVisitorMap::All(&self.cx.tcx.map)
+        NestedVisitorMap::None
```

Comment:
Because it never made sense to begin with. We'd be running the lint twice, once through the visitor and once through the LateLintPass